301-parameterized-linked-templates/nested/paramvm.json

```diff
@@ -0,0 +1,208 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "metadata": {
+                "description": "Azure region for VM"
+            }
+        },
+        "vm_name": {
+            "defaultValue": "",
+            "type": "string"
+        },
+        "vm_size": {
+            "defaultValue": "Standard_D8s_v3",
```

Comment:
remove defaultValues to the params that need to be passed through

Reply:
fixed